docs/pages/components/ClickOutsideListener.mdx

```diff
@@ -13,7 +13,7 @@ import { ClickOutsideListener } from '@deque/cauldron-react'
 ## Example
 
 ```jsx example
-<ClickOutsideListener onClickOutside={() => alert('You clicked outside!')}>
+<ClickOutsideListener onClickOutside={() => console.log('You clicked outside!')}>
```

Comment:
The `alert` was preventing navigation from happening. I don't think this being an alert is important enough to keep in considering we're just using this for documentation purposes.